analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/SymbolicContextTest.cs

```diff
@@ -68,20 +69,20 @@ public void CreateSymbolicValue_UsesSymbolicValueCounter()
         }
 
         [TestMethod]
-        public void SetOperationConstraint_WithExistingValue_ReusesProgramState()
+        public void SetOperationConstraint_WithExistingValue()
```

Comment:
For clarity: `SetOperationConstraint_WithExistingValue_CreatesNewState`

Reply:
Should have same destiny as conversion below